public/app/features/explore/Explore.tsx

```diff
@@ -194,12 +187,13 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
 
   renderGraphPanel(width: number) {
     const { graphResult, absoluteRange, timeZone, splitOpen, queryResponse, loading, theme } = this.props;
+    const spacing = parseInt(theme.spacing(1).slice(0, -2), 10);
     return (
       <Collapse label="Graph" loading={loading} isOpen>
         <ExploreGraphNGPanel
           data={graphResult!}
           height={400}
-          width={width - theme.panelPadding * 2}
+          width={width - spacing * 2}
```

Comment:
Can't we just do `width={width - theme.spacing(2)}`?

Reply:
But theme.spacing(2) returns 16px. Which is the same as 
`const spacing = parseInt(theme.spacing(1).slice(0, -2), 10);` => `8px`
`spacing * 2` => `16px`

Reply:
Oh yeah, you're right! But we still need to slice it right? Since `theme.spacing(2)` is a string and `width` is an int?

Reply:
Yes, you are correct.